### DISCUSSION TITLE
VisiData for data entry - Moving to the next cell

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1321] [Q&A] [answered]
I have started using VisiData for data entry. But one annoying thing is that after typing `e` to edit a cell `return` stays on teh same cell.

Is there a way after editing automatically progress to the next cell (either to the right or down)? For data entry the best would be to even go into edit mode after progressing to the next cell...

**Comment by saulpw** [comment, marked as answer]
Hi @halloleo, you should be able to use Shift+Arrow to move to the next cell and keep editing.

**Reply by halloleo** [reply]
Cool. Works exactly as I need it.

Thanks for telling me here. Must have missed it in the tutorial.

**Reply by anjakefala** [reply]
In v2.0, there is also an `options.cmd_after_edit` which allows the setting for the default longname executed after editing. For e.g. you can set the option to `go-right`, and after an edit terminates, it will default to the right. This got removed in v2.1!

**Reply by halloleo** [reply]
> In v2.0, there is also an `options.cmd_after_edit` ...  This got removed in v2.1!

Just curious: Why got it again removed?

**Reply by anjakefala** [reply]
It effectively got replaced by the Shift+Arrow: 98d4911c93163fc59123c27ba66959b4b13ed74b. Which is a more dynamic solution.

So, pre-2.1, the recommended thing to use is `options.cmd_after_edit`. Post-2.1, the recommendation is Shift+Arrow.
